packages/expo-updates/expo-updates-gradle-plugin/src/main/kotlin/expo/modules/updates/ExpoUpdatesPlugin.kt

```diff
@@ -26,7 +27,7 @@ abstract class ExpoUpdatesPlugin : Plugin<Project> {
     val entryFile = detectedEntryFile(reactExtension)
     val androidComponents = project.extensions.getByType(AndroidComponentsExtension::class.java)
 
-    if (System.getenv("EX_UPDATES_NATIVE_DEBUG") == "1") {
+    if (isNativeDebuggingEnabled(reactExtension)) {
```

Comment:
nit: there's `project` at line 22 so you can just pass project into the function. it's little redundant to use reactExtension here

Reply:
Fixed this in the next PR